storages/backends/google_cloud.py

```diff
@@ -0,0 +1,68 @@
+import base64
+import json
+from datetime import datetime
+from typing import AnyStr
+
+from google.cloud import storage, exceptions  # type: ignore
```

Comment:
I was getting an error `storages/backends/google_cloud.py:6: error: Module "google.cloud" has no attribute "storage"` so I've decided to put here ignore because changing version or reinstall didn't help.

Reply:
Where were you getting this error? In the IDE or did it fail the linting process? If it's the former we can probably ignore it.

Reply:
from mypy